Module.php

```diff
@@ -27,6 +27,6 @@ public function onBootstrap(\Zend\Mvc\MvcEvent $e)
         $em  = $app->getEventManager();
         $sm  = $app->getServiceManager();
 
-        $em->attachAggregate($sm->get('ZF\HttpCache\HttpCacheListener'));
+        $sm->get(\ZF\HttpCache\HttpCacheListener::class)->attach($em);
     }
```

Comment:
You're already in the `ZF\HttpCache` namespace, so you don't need to globally qualify this. :smile: